### PR TITLE
Added code to Add and remove elements from ArrayList

### PR DESCRIPTION
**Pull request by deepakghadge19**
This is for coding practice for Array List

### CODE REVIEW

File: src/controlFlowCodingPractice_15March2023/ArrayListFeature.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package controlFlowCodingPractice_15March2023;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayListFeature {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		List<Integer> arrayList = new ArrayList<>();
```
Comment (amitsahu8) [inline src/controlFlowCodingPractice_15March2023/ArrayListFeature.java:10]: Why Integer and not int? explore this

File: src/controlFlowCodingPractice_15March2023/ArrayListFeature.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		for (int i = 0; i <= 5; i++) {
+			arrayList.add(i);
+		}
+		System.out.println("Array List :" + arrayList);
+		// Remove elements from ArrayList
+		arrayList.remove(3);
```
Comment (amitsahu8) [inline src/controlFlowCodingPractice_15March2023/ArrayListFeature.java:16]: check the element is removed or not

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/controlFlowCodingPractice_15March2023/ArrayListFeature.java [hunk 3]
```diff
@@ -4,23 +4,43 @@
...
-		// Print all elements from ArrayLis
-		for (int num : arrayList) {
-			System.out.print(num + " ");
-		}
+        // Remove elements from ArrayList
+        System.out.println("\nElement removed from arrayList is :" + arrayList.remove(1));
 
-	}
+        System.out.println("\nSize of array list after remove elements :" + arrayList.size());
+        System.out.println("Array list after remove element :" + arrayList);
```
Comment (amitsahu8) [inline src/controlFlowCodingPractice_15March2023/ArrayListFeature.java:21]: you are not checking the element is removed or not

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/controlFlowCodingPractice_15March2023/ArrayListFeature.java (hunk 3 above)
Comment (deepakghadge19) [reply src/controlFlowCodingPractice_15March2023/ArrayListFeature.java]: Done

File: src/controlFlowCodingPractice_15March2023/ArrayListFeature.java (hunk 2 above)
Comment (deepakghadge19) [reply src/controlFlowCodingPractice_15March2023/ArrayListFeature.java]: Done

File: src/controlFlowCodingPractice_15March2023/ArrayListFeature.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+public class ArrayListFeature {
+    public void arrayListOperation() {
+        List<String> arrayList = new ArrayList<>();
+        arrayList.add("deepak");
+        arrayList.add("suyog");
+        arrayList.add("nikhil");
+        arrayList.add("dipesh");
+        System.out.println("Size of array list before remove element :" + arrayList.size());
+        System.out.println("Array list before remove element :" + arrayList);
+
```
Comment (amitsahu8) [inline src/controlFlowCodingPractice_15March2023/ArrayListFeature.java:15]: remove .class file

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/controlFlowCodingPractice_15March2023/ArrayListFeature.java (hunk 4 above)
Comment (deepakghadge19) [reply src/controlFlowCodingPractice_15March2023/ArrayListFeature.java:15]: .class file is removed

**Review by rohitlamsonge66 — APPROVED** [review]

**Pull request merged**